docs/docsite/rst/mazer/collection.rst

```diff
@@ -0,0 +1,169 @@
+
+Common fields
+-------------
+
+`name`
+^^^^^^
+
+A collection name.
+
+`version`
+^^^^^^^^^
+
+A collection version. Galaxy uses `Semantic Versioning <https://semver.org/>`_
+for collection versions, so you should follow some basic rules:
+
+* Use version numbers with three numeric parts such as `1.0.0` rather than `1.0`.
+
+`authors`
+^^^^^^^^^
+
+The list of collection authors. Each author is a string with a name and
+optionally email and url:
+
+.. code-block:: yaml
+
+  authors:
+    - "Alice Williams <awilliams@example.com>"
+    - "Bob Smith (http://bsmith.example.com)"
+    - "Jack Jackson"
+
+`description`
+^^^^^^^^^^^^^
+
+A short summary description of the collection.
+
+`license`
+^^^^^^^^^
+
+A license of a collection. If you are using a common license such as
+3-Clause BSD or MIT, add a current SPDX license identifier for the license
+you are using:
+
+.. code-block:: yaml
+
+  license: "MIT"
+
+You can use `The SPDX License List <https://spdx.org/licenses/>`_.
+`OSI Approvded <https://opensource.org/licenses/alphabetical>`_ licenses
+are more preferrable.
+
+keywords
+^^^^^^^^
+
+A list of collection keywords.
+
+Dependencies
+^^^^^^^^^^^^
+
+A collection may depend on other collections. Dependencies are specified
+in `dependencies` section as a mapping between collection name and a
+version string. The version string is a semver version spec:
+
+.. code-block:: yaml
+
+  dependencies:
+    geerlinguy.java: "1.3.0"
+
+Content
+^^^^^^^
+
+Usually all content units (e.g. roles, modules) that are located in standard
+directories included into collection build. You can customize this behaviour
+in `content` section of the manifest file:
+
+.. code-block:: yaml
+
+  content:
+    roles:
+      - "ansible/roles/*"
+    modules:
+      - "ansible/lib/*"
+    # ...
+
+`MANIFEST.json`
+===============
+
+The `MANIFEST.json` file is required to be located at the root level of
+collection artifact. It is automatically generated by `mazer` tool
+and contains collection manifest attributes and additional information
+about collection structure and its contents.
+
+Whereas `galaxy.yml` contains manifest in human friednly format and allows
+multiple fields formats such as simplified dependency definition syntax,
+`MANIFEST.json` should is used by galaxy server to import package and
+should be explicit and unambiguous:
+
+.. code-block:: json
+
+  {
+    "_format": 1,
+    "name": "geerlinguy.jenkins",
+    "version": "3.5.0",
+    "authors": [
+      { "name": "Jeff Geerling", "email": null, "url": null }
+    ],
+    "keywords": ["development", "packaging", "jenkins", "ci"],
+    "description": "Install and configure Jenkins CI server",
+    "license": "(BSD-3-Clause OR MIT)",
+    "license_file": null,
+    "readme": "README.md",
+
+    "dependencies": {
+      "geerlinguy.java": {"version": "1.3.0"},
+      "geerlinguy.nginx": {
+        "git": "http://github.com/geerlinguy/ansible-role-nginx.git",
+        "branch": null
+      }
+    },
```

Comment:
The above block should be formatted like this:

```
"collection_info": {
        "description": "testing some foo",
        "license": "GPL-3.0-or-later",
        "readme": "README.md",
        "version": "0.0.1",
        "dependencies": [],
        "authors": [
            "chouseknecht"
        ],
        "keywords": [
            "foo",
            "bar",
            "baz"
        ],
        "name": "foo.foo"
    },
    "format_version": 0.0
```

Reply:
Also, please change '_format' to 'format', as the '_' represents a private attribute that can't be set in a Python model, when using `attrs`

Reply:
I renamed `_format` to `format`. Keeping MANIFEST structure as is, as a result of today's call.